docker-compose.yml

```diff
@@ -6,7 +6,7 @@ services:
     container_name: 'ecamp3-frontend'
     ports:
       - '3000:3000'
-      - '9229:9229' # jest debug 
+      - '9229:9229' # jest debug
```

Comment:
Please revert your changes to this file. You can also instruct Git to ignore all your changes to this file:
```bash
git update-index --assume-unchanged docker-compose.yml
```
But this does not survive branch switching, and sometimes it's also more confusing than it helps.
To revert this git mind control:
```bash
git update-index --no-assume-unchanged docker-compose.yml
```

Reply:
This is quite annoying, because without these changes (I did not want to commit) it doesn't work for me 😕

Reply:
Maybe I can rely on the different PhpStorm changelists